build.gradle

```diff
@@ -10,7 +10,6 @@ buildscript {
 		classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
 		classpath "org.kt3k.gradle.plugin:coveralls-gradle-plugin:2.8.2"
 		classpath "org.jetbrains.dokka:dokka-gradle-plugin:0.9.18"
-		classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
```

Comment:
I can split this out, and originally meant to just do the action portion, but this repo doesn't get much attention and the PR is small otherwise